src/providerhub/azext_providerhub/generated/_params.py

```diff
@@ -291,54 +356,54 @@ def load_arguments(self, _):
-                                                             'UnboundedClientGroupMembership']), help='ARM allows customized headers when sending requests to the RP. This can be done both at the provider level or at the individual resource type level.',
+                                                             'UnboundedClientGroupMembership']), help='',
                    arg_group='Request Header Options')
-        c.argument('subscription_state_rules', action=AddSubscriptionStateRules,
-                   nargs='+', help='The subscription policy.')
-        c.argument('template_deployment_options', action=AddTemplateDeploymentOptions,
-                   nargs='+', help='The field for preflight options.')
-        c.argument('extended_locations', action=AddExtendedLocations,
-                   nargs='+', help='The extended locations property.')
-        c.argument('resource_move_policy', action=AddResourceMovePolicy, nargs='+',
-                   help='Indicates the resource type has opted in to move operations.')
-        c.argument('resource_deletion_policy', arg_type=get_enum_type(['NotSpecified', 'CascadeDeleteAll',
-                                                                       'CascadeDeleteProxyOnlyChildren']), help='The property to customize RPaaS deletion operation.')
+        c.argument('subscription_state_override_actions', action=AddSubscriptionStateOverrideActions, nargs='+',
+                   help='', arg_group='Subscription Lifecycle Notification Specifications')
+        c.argument('soft_delete_ttl', help='',
+                   arg_group='Subscription Lifecycle Notification Specifications')
+        c.argument('required_features_policy', arg_type=get_enum_type(['Any', 'All']), help='', arg_group='Features '
+                   'Rule')
+        c.argument('resource_creation_begin', action=AddExtensionOptions, nargs='+', help='',
+                   arg_group='Extension Options')
+        c.argument('resource_patch_begin', action=AddExtensionOptions, nargs='+', help='',
+                   arg_group='Extension Options')
 
     with self.argument_context('providerhub resource-type-registration update') as c:
         c.argument('provider_namespace', type=str,
                    help='The name of the resource provider hosted within ProviderHub.')
         c.argument('resource_type', type=str, help='The resource type.')
-        c.argument('routing_type', arg_type=get_enum_type(['Default', 'ProxyOnly', 'HostBased', 'Extension',
-                                                           'Tenant', 'Fanout', 'LocationBased', 'Failover',
-                                                           'CascadeExtension']), help='The resource routing type.')
+        c.argument('nested_resource_type', type=str,
+                   help='The nested resource type.', id_part='child_name_2')
+        c.argument('routing_type', type=str, help='The resource routing type.')
```

Comment:
It's a breaking change on command line interface.

Reply:
removed nested_resource_type